libnopegl/src/pgcraft.c

```diff
@@ -41,7 +41,7 @@
 #if defined(BACKEND_GL) || defined(BACKEND_GLES)
 #include "backends/gl/gpu_ctx_gl.h"
 #include "backends/gl/feature_gl.h"
-#include "backends/gl/program_gl_utils.h"
+#include "backends/gl/gpu_program_gl_utils.h"
```

Comment:
doesn't this belong to an earlier commit?

Reply:
I can squash it with an earlier commit.

Reply:
It belongs to the right commit. This module was forgotten in the previous PR (adding the gpu_ prefix).